docs/user_manual/yaydoc_configuration.md

```diff
@@ -0,0 +1,135 @@
+| source           | This is the path which would be scanned for markdown and reST files. Also any static content such as images referenced from embedded html in markdown should be placed under a _static directory inside source. Note that the README would always be included in the starting page irrespective of source from the auto-generated index | docs/ |
+| logo             | The path to an image to be used as logo for the project. The path specified should be relative to the source directory. | _none_ |
+| markdown_flavour | The markdown flavour which should be used to parse the markdown documents. Possible values for this are markdown, markdown_strict, markdown_phpextra, markdown_github, markdown_mmd and commonmark. Note that currently this option is only used for parsing any included documents using the mdinclude directive and it's unlikely to change soon. | markdown_github |
+| mock             | Any python modules or packages which should be mocked. This only makes sense if the project is in python and uses autodoc has C dependencies. | _none_ |
+| autoapi          | If enabled, Yaydoc will crawl your repository and try to extract API documentation. Provides attributes for specifying the language and source path. Currently supported languages are java and python | _none_ |
+| subproject       | This section can be used to include other repositories when building the docs for the current repositories. The source attribute should be set accordingly. | _none_ |
+| github_button    | This section can be used to include various Github buttons such as fork, watch, star, etc. | hidden |
+| github_ribbon    | This section can be used to include a Github ribbon linking to your github repo. | hidden |
+<!-- endmarkdown+ -->
+
+```yaml
+build:
+  theme: 
+    name: sphinx_fossasia_theme
+  source: docs
+  logo: images/logo.svg
+  markdown_flavour: markdown_github
+  mock:
+    - numpy
+    - scipy
+  autoapi:
+    - language: python
+      source: modules
+    - language: java
+  subproject:
+    - url: <URL of Subproject 1>
+      source: doc
+    - url: <URL of subproject 2>
+  github_ribbon:
+    position: right
+    color: green
+  github_button:
+    buttons:
+      watch: true
+      star: true
+      issues: true
+      fork: true
+      follow: true
+    show_count: true
+    large: true
+```
+
+## publish
+Following is a description of config options under the `publish` section and it's example usage.
+
+<!-- markdown+ -->
+| Key | Description | Default |
+|-----|-------------|---------|
+| ghpages | It provides a attribute url whose value is written in a CNAME file while publishing to github pages. | _none_ |
+| heroku  | It provides an app_name attribute which is used as the name of the heroku app. Your docs would be deployed at <app_name>.herokuapp.com | _none_ |
```

Comment:
`<app_name>` isn't rendering. Only `.herokuapp.com` is displayed.

Reply:
Updated